scripts/o3de/ExportScripts/export_source_built_project.py

```diff
@@ -86,12 +97,17 @@ def export_standalone_project(ctx: exp.O3DEScriptExportContext,
     # Resolve (if possible) and validate any provided seedlist files
     validated_seedslist_paths = exp.validate_project_artifact_paths(project_path=ctx.project_path,
                                                                     artifact_paths=seedlist_paths)
+    
+    # Convert level names into seed files that the asset bundler can utilize for packaging
+    for level in level_names:
+        seedfile_paths.append(str(pathlib.PurePath('levels') / level.lower() / (level.lower() + ".spawnable")))
```

Comment:
As levels are in the project folder, we should append this path to the `ctx.project_path`, so the Asset Bundler gets an absolute path to the levels.
```suggestion
        seedfile_paths.append(ctx.project_path / pathlib.PurePath('levels') / level.lower() / (level.lower() + ".spawnable")))
```